test/Ocelot.AcceptanceTests/PathToDownstreamUrlTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using IdentityServer4.AccessTokenValidation;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.AcceptanceTests;
+
+public class PathToDownstreamUrlTests : IDisposable
```

Comment:
Can we move the test to more appropriate class which is `RoutingTests` in namespace `Ocelot.AcceptanceTests.Routing`?
We keep all routing vs path tests in this class.